x-pack/plugins/enterprise_search/public/applications/workplace_search/views/account_settings/account_settings.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useEffect, useMemo } from 'react';
+
+import { useValues } from 'kea';
+
+import type { AuthenticatedUser } from '../../../../../../security/public';
+import { KibanaLogic } from '../../../shared/kibana/kibana_logic';
+
+export const AccountSettings: React.FC = () => {
+  const { security } = useValues(KibanaLogic);
+
+  const [currentUser, setCurrentUser] = useState<AuthenticatedUser | null>(null);
+
+  useEffect(() => {
+    security!.authc!.getCurrentUser().then(setCurrentUser);
```

Comment:
__question:__ won't we have an unhandled exception here in case security plugin is disabled?

Reply:
Thanks for checking our plugin code as well! Good question.

No, if this component loads, we are guaranteed to have the security plugin enabled. 
One of the requirements for being able to run Workplace Search is to have security enabled (see Step 2 [in our installation docs](https://www.elastic.co/guide/en/workplace-search/current/workplace-search-install.html#installation-self-managed)). If security is disabled, Workplace Search won't run and this page could not be opened.

Reply:
Got it, thanks for clarifying! Was just confused by these `!` and was wondering if `security` and its properties can really be `undefined` or not (by the way it seems `!` isn't strictly needed for `security` itself and is only needed to access its properties).

Reply:
You're right. I guess I overdid it here. 😄 
I plan to do a follow-up PR with some cleanup in the enterprise_search plugin only.
I'll update this line there if you don't mind.

Reply:
Sounds good to me :+1: